ta/pkcs11/include/pkcs11_ta.h

```diff
@@ -900,7 +900,7 @@ enum pkcs11_user_type {
  * Once TEE Identity based authentication is activated following operational
  * changes happen:
  * - PIN failure counters are disabled to prevent token authentication lockups
- * - Switching to different authentication mode needs C_InitToken()
+ * - Switching to different authentication mode is not possible
```

Comment:
Isn't that a bit annoying? How about entering the TEE identity auth with a special PIN format rather than a NULL/empty SO PIN? For instance: "teeauth:<PIN>". Then allow switching back to the usual mode with a "change SO PIN" operation that would require the same SO PIN?

Reply:
Please see the next commit.